Directory.Build.targets

```diff
@@ -34,4 +44,26 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsTestProject)' != 'true' and '$(ExcludeFromAssemblyMetadata)' != 'true'" >
```

Comment:
```suggestion
  <ItemGroup Condition="'$(IsSourceProject)' == 'true' and '$(ExcludeFromAssemblyMetadata)' != 'true'" >
```

Reply:
I think this new property is just adding unnecessary complexity. I added to try to skip Versions.csproj from getting the CLSCompliant property set. I think I can just explicitly mark Versions.csproj as "IsSourceProject=false" and be done.

Reply:
Yep. Just wanted to mention this too.